x/wasmplus/types/tx_test.go

```diff
@@ -3,6 +3,7 @@ package types
 import (
 	"encoding/hex"
 	"fmt"
+	"github.com/line/lbm-sdk/x/auth/legacy/legacytx"
```

Comment:
Please keep the rule import order. 

1. go default library like fmt.
2. external library not line
3. other line library, ex: github.com/line/tm-db
4. ostracon itself library

Reply:
Thank you, I didn't know abount this rule.

I have a question about the order of imports. Do I need to set order of imports by hand?
even though I ran `make lint` and `make format`, I didn't get the same order of imports.
below is what I got after run `make lint` and `make format`.
```
import (
	"encoding/hex"
	"fmt"
	"testing"

	"github.com/line/lbm-sdk/x/auth/legacy/legacytx"

	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"

	sdk "github.com/line/lbm-sdk/types"

	wasmTypes "github.com/line/wasmd/x/wasm/types"
)
```

Reply:
Yes, we need to sort manually.